packages/spor-react/src/theme/slot-recipes/checkbox.ts

```diff
@@ -2,38 +2,56 @@ import { coreBackground, coreText } from "../utils/core-utils";
 import { brandBackground, brandText } from "../utils/brand-utils";
 import { focusVisibleStyles } from "../utils/focus-utils";
 import { defineSlotRecipe } from "@chakra-ui/react";
+import { checkboxAnatomy } from "./anatomy";
 
 export const checkboxSlotRecipe = defineSlotRecipe({
-  slots: ["control", "label", "root", "icon"],
   className: "spor-checkbox",
+  slots: checkboxAnatomy.keys(),
   base: {
     root: {
+      display: "flex",
+      gap: 1.5,
       _hover: {
-        ...coreBackground("hover"),
-        borderColor: brandBackground("hover").backgroundColor,
+        "& > input:enabled:not([aria-invalid]) + .spor-checkbox__control": {
+          ...coreBackground("hover"),
+          borderColor: brandBackground("hover").backgroundColor,
+        },
+        "& > input:enabled:checked:not([aria-invalid]) + .spor-checkbox__control":
+          {
+            ...brandBackground("hover"),
+            borderColor: brandBackground("hover").backgroundColor,
+          },
       },
-      _invalid: {
-        backgroundColor: "white",
-        borderColor: "outline.error",
+    },
+    indicator: {
+      width: "100%",
+      height: "100%",
+      borderColor: "white",
+      marginTop: -1,
+      transform: "scale(1)",
+      opacity: 0,
+      "& polyline": {
+        fill: "none",
+        transformOrigin: "50% 50%",
+        strokeWidth: "3px",
+        strokeDasharray: "22.771367900227325",
```

Comment:
ble nysgjerrig på dette verdi og merket at det har ingen effekt med eller uten, ihvertfall for de eksempler som er på selve side. Er det et forsøkt å få animation på plass?